services/apps/alcs/src/alcs/search/search.controller.spec.ts

```diff
@@ -346,7 +346,7 @@ describe('SearchController', () => {
 
     expect(
```

Comment:
suggestion: Now that the test is focused on Non Applications, you should remove the Notice of Intent expects as they are not pertinent.